app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -1057,11 +1057,19 @@ public static void bgReadingInsertFromInt(int value, long timestamp, boolean do_
 
             bgr.timestamp = timestamp;
             bgr.calculated_value = value;
+            
 
             // rough code for testing!
             bgr.filtered_calculated_value = value;
-            bgr.raw_data = value*1000;
```

Comment:
Wont this break all existing calibrations from LibreAlarm etc?

Reply:
bgReadingInsertFromInt is only called from the function CalculateFromDataTransferObject and only if use_raw==false.

For existing calibrations from LibreAlarm there is no way (and there wasn't before my change) to have use_raw==false.
So, the function CalculateFromDataTransferObject () was never called before my change, so I thought that changing it is save.

Reply:
ok great